upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,19 @@
 == Migration Changes
 
+=== Migrating to 8.0.1
+
+==== More Authentication flows changes
+
+REQUIRED and ALTERNATIVE executions not supported at same flow::
+    In previous version, it was possible to have REQUIRED and ALTERNATIVE executions in the same authentication flow at the same level.
+    There were some issues with this approach and we did the refactoring in the Authentication SPI, which means that this is not considered
+    valid anymore. If ALTERNATIVE and REQUIRED executions are configured at the same level, the ALTERNATIVE executions are considered disabled.
+    So when migrating to the newest version, your existing authentication flows will be automatically migrated to preserve same behaviour as was in previous version.
+    If they contain ALTERNATIVE executions after some REQUIRED executions, then the ALTERNATIVE executions will be added to the separate REQUIRED subflow. This
+    should ensure same/similar behaviour of the particular authentication flow as in previous version. However it is always recommended
```

Comment:
use US spelling for words ending in "our" - such as color instead or colour and behavior instead of behaviour.